legaltext/models.py

```diff
@@ -144,3 +146,14 @@ def anchor_callback(match):
             normalize_newlines(self.content).replace('\n', '<br />')
         )
         return render_markdown(content).replace('<p>', '').replace('</p>', '')
+
+    def _get_next_checkbox_numeral(self):
+        last_checkbox = self.legaltext_version.checkboxes.last()
```

Comment:
I suppose a different approach, choose what you prefer: (self.legaltext_version.checkboxes.aggregate(next_order=Max('order'))['next_order'] or 0) + 1

Reply:
Cool! Done!